client/js/lounge.js

```diff
@@ -1331,7 +1331,7 @@ $(function() {
 					} else {
 						var current = $(this);
 
-						if (current.attr("class") === "date-marker") {
+						if (current.hasClass("date-marker") && prev.hasClass("date-marker")) {
```

Comment:
Simplest thing I can think of is changing the first one.

Currently we do:

```
if (!prev) {
 	// Should always be a date-seperator, because it's always added
 	prev = $(this);
} else {
...
```

So, if we want to make sure that prev is always a date-marker, we can just do this:

```
if (!prev) {
	// Should always be a date-seperator, because it's always added
	if ($(this).hasClass("date-marker")) {
	 	prev = $(this);
	}
} else {
...
```

Then you don't need to add this extra check for the date-marker on the previous one here, as we will always have a date marker on this one.